R/broom_wrapper.R

```diff
@@ -82,28 +82,41 @@ augment_kmeans <- function(df, model, data){
 #' @param model_df Data frame that has model
 #' @param ... Additional argument to be passed to broom::augment
 #' @export
-add_prediction <- function(df, model_df, ...){
+add_prediction <- function(df, model_df, conf_int = 0.95, ...){
   # parsing arguments of add_prediction and getting optional arguemnt for augment in ...
   cll <- match.call()
   aug_args <- expand_args(cll, exclude = c("df", "model_df"))
 
-  get_result_with_response <- function(model_df, df, aug_args){
+  get_result <- function(model_df, df, aug_args, with_respose){
     # Use formula to support expanded aug_args (especially for type.predict for logistic regression)
     # because ... can't be passed to a function inside mutate directly.
     aug_fml <- if(aug_args == ""){
       as.formula("~list(broom::augment(model, newdata = df))")
     } else {
       as.formula(paste0("~list(broom::augment(model, newdata = df, ", aug_args, "))"))
     }
-    model_df %>%
+    ret <- model_df %>%
       # result of aug_fml will be stored in .text_index column
       # .test_index is used because model_df has it and won't be used here
       dplyr::mutate_(.dots = list(.test_index = aug_fml)) %>%
-      dplyr::ungroup() %>%
-      dplyr::mutate(.test_index = purrr::map2(.test_index, model, function(d, m){
-        # add predicted_response to the result data frame
-        add_response(d, m, "predicted_response")
-      })) %>%
+      dplyr::ungroup()
+    ret <- if(with_respose) {
+     ret %>%
```

Comment:
The following line has only one space for its indentation. Is this intentional?
ret %>%